### PR TITLE
simplify and unify texts

### PR DESCRIPTION
**Pull request by rpopov**
Avoid referring same things with different terms. Provide a description from the build, through stages, jobs down to the phases. Avoid referring same things with different terms. Provide a description from the build, through stages, jobs down to the phases.
Put the terms in their context

### CODE REVIEW

File: user/job-lifecycle.md [hunk 1]
```diff
@@ -9,19 +9,27 @@ Travis CI provides a default build environment and a default set of phases for e
 
 Keep reading to see how you can customize any phase in this process, via your `.travis.yml`.
 
+## The build
+
+The `.travis.yml` file describes the build (process. A *build* in Travis CI is a sequence of [stages](../for-beginners/#builds-jobs-stages-and-phases). Each *stage* consists of [jobs](../for-beginners/#builds-jobs-stages-and-phases) run in parallel.
```
Comment (noloader) [inline user/job-lifecycle.md]: Drop the parens. The parent that splits one sentence and continues into another sentence really breaks flow.

Also, the statement in parens are like whispers. I don't think you should whisper what a build is. You should clearly state it.  In contrast, dashes - like this - is like yelling. Dashes are intended to make something really stand out. In general, don't whisper and don't yell. Write in short sentences like you would talk.

File: user/job-lifecycle.md [hunk 2]
```diff
@@ -9,19 +9,27 @@ Travis CI provides a default build environment and a default set of phases for e
...
+1. `after_script` - after the script phase.
+1. `after_success` - when the build *succeeds* (e.g. building documentation), the result is in `TRAVIS_TEST_RESULT` environment variable
+1. `after_failure` - when the build *fails* (e.g. uploading log files), the result is in `TRAVIS_TEST_RESULT` environment variable
 
-You can perform additional actions when your build succeeds or fails using the `after_success` (such as building documentation) or `after_failure` (such as uploading log files) phases.
-In both `after_failure` and `after_success`, you can access the build result using the `$TRAVIS_TEST_RESULT` environment variable.
+There are three optional *deployment phases*.
 
-The complete job lifecycle, including three optional deployment phases and after checking out the git repository and changing to the repository directory, is:
+The complete sequence of phases (lifecycle) of a job is:
```
Comment (noloader) [inline user/job-lifecycle.md]: The complete sequence of phases of a job is the lifecycle. The steps are:

File: user/job-lifecycle.md (hunk 1 above)
Comment (Pezi777) [inline user/job-lifecycle.md]:
```suggestion
The `.travis.yml` file describes the build process. A *build* in Travis CI is a sequence of [stages](../for-beginners/#builds-jobs-stages-and-phases). Each *stage* consists of [jobs](../for-beginners/#builds-jobs-stages-and-phases run in parallel. 
```

File: user/job-lifecycle.md (hunk 2 above)
Comment (Pezi777) [inline user/job-lifecycle.md]:
```suggestion
The complete sequence of phases of a job is the lifecycle. The steps are:
```

**Comment by Sa1Beer777** [comment]
> Avoid referring same things with different terms. Provide a description from the build, through stages, jobs down to the phases. Avoid referring same things with different terms. Provide a description from the build, through stages, jobs down to the phases.
> Put the terms in their context

rpopov:patch-2

**Pull request merged**